content/09.evolution.md

```diff
@@ -96,11 +96,45 @@ IFNs and Ribavirin have shown promise in _in vitro_ analyses of MERS, but their
+
+Nevertheless, some fragments of the genome differ between SARS-CoV-2 and RATG13 by up to 17%, suggesting a complex natural selection process.
+Additionally, SARS-CoV-2 is closely related (91.02%) to a novel coronaviruses identified in Malayan pangolins (_Manis javanica_) infected with a respiratory disease in October 2019 [@doi:10.1016/j.cub.2020.03.022].
+Although the genome-wide sequence identity was lower between SARS-CoV-2 and this pangolin virus than BatCoV-RaTG13, its particularly high similarity in the receptor binding domain (RBD) of the spike (_S_) gene with SARS-CoV-2 drew further attention [@doi:10.1038/s41586-020-2169-0; @doi:10.1016/j.cub.2020.03.022].
+The SARS-CoV-2 RBD differs from the pangolin coronavirus RBD by only one amino acid change [@doi:10.1016/j.cub.2020.03.022], and the sequence identity between the regions is 97.4% [@doi:10.1038/s41586-020-2169-0].
+Pangolins were therefore identified as a potential intermediate host of SARS-CoV-2 between bats and humans.
+
+However, data collected from May 2017 to November 2019 by a research team interested in tick-borne illnesses identified no bats or pangolins sold at these markets leading up to the emergence of COVID-19 [@doi:10.1038/s41598-021-91470-2].
+Therefore, it is not clear whether the disease emerged in a different location and/or whether it is associated with a different species.
+There were 38 species observed at the market in the 2.5 years leading up to the emergence of SARS-CoV-2, indiciating significant diversity in the animals with which humans were interacting [@doi:10.1038/s41598-021-91470-2].
+As with SARS-CoV-1, the species of origin for SARS-CoV-2 therefore remains unresolved.
+
+Genomic analyses and comparisons to other known coronaviruses suggest that SARS-CoV-2 is unlikely to have originated in a laboratory -- either purposely engineered and released, or escaped -- and instead evolved naturally in an animal host [@doi:10.1038/s41591-020-0820-9].
+However, potentially due to public misunderstanding about recombination and complex evolutionary processes like coevolution, the similarity to pangolin _S_ has resulted in popular conspiracy theories that the virus did not arise naturally.
+The similarity of _S_ to that of pangolin viruses could arise from either recombination or coevolution [@doi:10.1093/nsr/nwaa036; @doi:10.1126/sciadv.abb9153], rather than requiring human intervention.
+Such suspcions may also have been fueled, in part, by the lack of well-characterized bat coronaviruses which means that SARS-CoV-2 is still relatively derived from documented coronaviruses surveyed in bats [@doi:10.1016/j.tree.2020.12.002].
+This ambiguity therefore serves to highlight the value of monitoring zoonotic diseases, as more robust reference panels against which to compare SARS-CoV-2 would be valuable in pinpointing its origin.
```

Comment:
I don't know what you think about this - but my thought is we shouldn't shy away from the fact there is political motivations here too to promote a lab leak theory,  that we don't need to delve into, but should briefly acknowledge.

Reply:
Maybe something like:

```suggestion
While critiques that more thorough investigation of the origins of COVID-19 would be valuable may have some merit, in many cases, claims made in support of the "lab-leak" theory are politically motivated [@doi:10.1038/d41586-021-01383-3].
A more robust panel of zoonotic viruses against which to compare SARS-CoV-2 would allow for conclusive dismissal of these politicized claims, underscoring another potential benefit of monitoring zoonotic diseases.
```

Reply:
Sorry I wasn't sure of the wording, you might want to edit my suggestion @rando2

Reply:
Did you mean critics?

Reply:
I made a few changes, what do you think of this? And I don't think I meant to say critics anywhere but I'm not quite sure what part you're referring to, sorry @RLordan!

Reply:
Good with me! No worries, just to let you know that I edited out the word "critiques" from the line that starts "While it has been suggested that more ..."